README.md

```diff
@@ -11,8 +11,54 @@
 
 ## Overview
 
+Autoshare for Twitter automatically tweets your posts as soon as they’re published.  Once you hit the Publish button, the plugin sends your post’s title, featured image, and link to Twitter, along with a custom message.
+
+| Create post screen with Autoshare for Twitter options | Published post screen with Autoshare for Twitter options. |
+| -------------- | ---------------- |
+| [![Create post screen with Autoshare for Twitter options](.wordpress-org/screenshot-1.png)](.wordpress-org/screenshot-1.png) | [![Published post screen with Autoshare for Twitter options.](.wordpress-org/screenshot-2.png)](.wordpress-org/screenshot-2.png) |
+
+Unlike a myriad of other social media, multitool solutions, Autoshare for Twitter is built solely for Twitter.  It focuses on doing one thing and does it well, with the code and interface craftsmanship we apply to every project.
+
+With Autoshare for Twitter, developers can further customize nearly everything about the tweets, including the image, author, and link, using an extensive set of hooks built into the code. Among its other features, the WordPress plugin:
+
+* Works in both the classic and new block editors.
+* Becomes part of the pre-publish checklist step that’s part of the new block editor.
+* Posts a high-quality featured image with your tweet.
+* Counts characters to keep you under the tweet limit.
+* Adds a link to the tweet in the block editor Status & Visibility panel.
```

Comment:
As we have a dedicated Panel for Autoshare for Twitter now and we are no longer using the Status & Visibility panel can we update this to something like this?

```suggestion
* Adds a link to the tweet in the block editor Autoshare for Twitter panel.
```
![image](https://github.com/10up/autoshare-for-twitter/assets/10613171/1c9df7b0-5d23-47d9-a2e4-1ef3022ed914)

Reply:
Based on the screenshot, looks like we should update to `Autotweet` instead of `Autoshare for Twitter`, correct?

Reply:
The title of the panel is dynamic and it updates based on status. So, not sure what to name it. 🤔 

![image](https://github.com/10up/autoshare-for-twitter/assets/10613171/76153288-ce11-4725-bcd5-5932b76b9231)
![image](https://github.com/10up/autoshare-for-twitter/assets/10613171/61526a23-cb21-44eb-aea6-61ebc0d8b795)
![image](https://github.com/10up/autoshare-for-twitter/assets/10613171/71a42939-3e42-4408-b47a-1f38256912da)

Reply:
OR Maybe better to make it more generic?
`* Adds a link to the tweet in the block editor sidebar`